### PR TITLE
Backports 0.16 pr13

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
Test failure likely unrelated see #3388

**Comment by PastaPastaPasta** [comment]
resolved all comments

**Comment by PastaPastaPasta** [comment]
rebased on develop to hopefully make travis happy

File: src/wallet/rpcdump.cpp [hunk 1]
```diff
@@ -780,7 +797,7 @@ UniValue dumpwallet(const JSONRPCRequest& request)
         throw std::runtime_error(
             "dumpwallet \"filename\"\n"
             "\nDumps all wallet keys in a human-readable format to a server-side file. This does not allow overwriting existing files.\n"
-            "Imported scripts are not currently included in wallet dumps, these must be backed up separately.\n"
+            "Imported scripts are included in the dumpfile, but corresponding BIP173 addresses, etc. may not be added automatically by importwallet.\n"
```
Comment (thephez) [inline src/wallet/rpcdump.cpp:800]: Segwit reference (BIP173)

**Pull request merged**